packages/tools/src/tools/annotation/SplineROITool.ts

```diff
@@ -724,25 +743,60 @@ class SplineROITool extends ContourSegmentationBaseTool {
     return true;
   }
 
+  /**
+   * Creates new interpolated handles for the spline control given the
+   * polyline data.  This allows creating the spline from polyline data
+   * directly.
+   */
+  protected createInterpolatedSplineControl(annotation) {
+    if (annotation.data.handles.points?.length) {
+      // The interpolation itself created the handles
+      return;
+    }
+    const { polyline } = annotation.data.contour;
```

Comment:
This code isn't actually active - I tested it, and it works, but it is the code in teh interpolation routine that is being used.  However, this one is required in case the spline comes from some other system which does not calculate handles.